application/gradle.properties

```diff
@@ -22,4 +22,6 @@ kotlin.code.style=official
 org.gradle.caching=true
 # TODO Return configureondemand on true after fix https://github.com/formatools/forma/issues/50
 org.gradle.configureondemand=false
-org.gradle.vfs.watch=true
\ No newline at end of file
+org.gradle.vfs.watch=true
+forma.kotlinVersion=1.5.10
```

Comment:
Just a suggestion. Should we use separate file for forma configuration?

Reply:
For now it doesn't metter, but if we find correct way to declare versions in single place using gradle api, we should not use custom solutions. Configuring plugins using gradle.properties looks like straight forward gradle way

Reply:
Having forma properties in the separate file(e.g forma.properties) will allow us to do automated refactorings(or generate file to new projects) more easily compare to file with all possible flags